projects/1.20/assets/enderite-cj-edition/enderite/lang/zh_cn.json

```
@@ -0,0 +1,106 @@
+{
+  "creativemodetab.enderite.enderite_tab": "末影合金",
+
+  "enchantment.enderite.night_vision": "夜视",
+  "enchantment.enderite.no_gravity": "无重力",
+  "enchantment.enderite.void_floating": "虚空漂浮",
+  "enchantment.enderite.night_vision.desc": "赋予玩家夜视效果",
+  "enchantment.enderite.no_gravity.desc": "使物品失重",
+  "enchantment.enderite.void_floating.desc": "保护物品不受虚空影响",
+
+  "item.enderite.endercrystal_shard": "末影水晶碎片",
+  "item.enderite.ender_dust": "末影尘",
+  "item.enderite.raw_enderite": "末影合金粗矿",
+  "item.enderite.enderite_scrap": "末影合金碎片",
+  "item.enderite.enderite_ingot": "末影合金锭",
+  "item.enderite.enderite_nugget": "末影合金粒",
+
+  "item.enderite.enderite_helmet": "末影合金头盔",
+  "item.enderite.enderite_chestplate": "末影合金胸甲",
+  "item.enderite.enderite_elytra_chestplate": "末影合金鞘翅盔甲",
+  "item.enderite.enderite_leggings": "末影合金护腿",
+  "item.enderite.enderite_boots": "末影合金靴子",
+
+  "item.enderite.enderite_sword": "末影合金剑",
+  "item.enderite.enderite_pickaxe": "末影合金稿",
+  "item.enderite.enderite_axe": "末影合金斧",
+  "item.enderite.enderite_shovel": "末影合金铲",
+  "item.enderite.enderite_hoe": "末影合金锄",
+  "item.enderite.ender_pearl_staff": "末影珍珠权杖",
+  "block.enderite.enderite_block": "末影合金块",
+  "block.enderite.enderite_ore": "末影合金矿",
+  "block.enderite.raw_enderite_block": "粗末影合金块",
+  "block.enderite.end_respawn_anchor": "末地重生锚",
+  "block.enderite.hardened_end_stone": "压缩末地石",
+  "block.enderite.budding_endercrystal": "末影水晶母岩",
+  "block.enderite.endercrystal_block": "末影水晶矿石",
+  "block.enderite.endercrystal_cluster": "末影水晶簇",
+  "block.enderite.large_endercrystal_bud": "大型末影水晶芽",
+  "block.enderite.medium_endercrystal_bud": "中型末影水晶芽",
+  "block.enderite.small_endercrystal_bud": "小型末影水晶芽",
+  "tooltip.enderite.only_creative": "仅限于创造模式",
+  "tooltip.enderite.enderite_sword_teleport": "把你传送到你所看到的地方",
+  "tooltip.enderite.enderite_sword_teleport_range": "区块范围",
+  "tooltip.enderite.enderite_sword_teleport_charges_info1": "请在你的物品清单中放入珍珠，然后按下鼠标右键进行充能",
```

inventory：物品栏
一般情况下第二人称可省略
原版术语问题请全文通查

```suggestion
  "tooltip.enderite.enderite_sword_teleport_charges_info1": "在物品栏中放入末影珍珠，然后按下鼠标右键进行充能",
```